testing/unittests/jlibtests.cpp

```diff
@@ -499,6 +499,11 @@ class JlibFileIOTestTiming : public CppUnit::TestFixture
                 extraFlags = IFEnocache;
             ifileio = ifile->open(IFOcreate, extraFlags);
 
+#if 0
+            if (server != NULL)
+                ifile->setShareMode((IFSHmode)IFSHfull);
+#endif
```

Comment:
I guess this is left in from Windows dafilesrv testing ?
Is there any reason not to leave it in, in this test in general ?

Reply:
I was testing with and without calling setShareMode().  Ideally it should be a cmd-line option to this test.